src/model_io/urdf/src/URDFGenericSensorsImport.cpp

```diff
@@ -71,7 +71,7 @@ bool findJointParentsAndChild(TiXmlElement* robotXml,
     for (TiXmlElement* joint_xml = robotXml->FirstChildElement("joint");
          joint_xml; joint_xml = joint_xml->NextSiblingElement("joint"))
     {
-        IJointPtr joint=0;
+        //IJointPtr joint=0;
```

Comment:
Feel free to remove this kind of "dead code", thanks! 
For storing history we have git, and leaving this kind of dead code commented out hanging around just complicated the readability.